### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by kangen22**
Added solution, worked in Node.js

### CODE REVIEW

**Review by MPTCourseTeacher — COMMENTED** [review]
I think you can move 'author' field and getter to the base class Book

**Comment by kangen22** [comment]
Refactor the code, review again :)

File: spr4.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+class Book {
+    constructor(title, _author) {
+        this.title = title; // public
```
Comment (MPTCourseTeacher) [inline spr4.js]: Don't make data fields public. Let's make it protected or private

**Review by MPTCourseTeacher — CHANGES_REQUESTED** [review]
Please, check my comment

File: spr4.js (hunk 1 above)
Comment (kangen22) [reply spr4.js]: Thanks for the review. I created public data because it was in the task

**Review by MPTCourseTeacher — APPROVED** [review]
Thank you for update. Now it looks much better :)

**Comment by kangen22** [comment]
Thanks for your review)

**Pull request merged**